app/policies/project_policy.rb

```diff
@@ -117,6 +117,22 @@ def transfer_sample_into_project?
     false
   end
 
+  def clone_sample?
+    return true if Member.user_has_namespace_maintainer_access?(user, record.namespace, false)
+    return true if Member.can_clone_sample?(user, record.namespace) == true
+
+    details[:name] = record.name
+    false
+  end
+
+  def clone_sample_into_project?
+    return true if Member.can_clone_sample_to_project?(user, record.namespace, false) == true
```

Comment:
Why do you have duplicate calls to `Member.can_clone_sample_to_project?` here? Maintainer/Owner can clone a sample to any project they have direct/inherited/grouplink membership in. The first line can be removed here

Reply:
I was referencing what we did for transfer samples and refactored it. Sorry, see 26d436d35e40ad191ed3b6a465ff7f3e9412c4f6.

Reply:
All good! :) Thanks for updating it